louiseA/file-1.js

```diff
@@ -0,0 +1,103 @@
+1.
+
+ a) "1" == 1
+  true
+
+ b) "1" === 1
+  false
+
+ c) x == "x"
+  true
+
+ d) x == (x+'')
+  false
+
+ e) '' == ' '
+  true
+
+ f) x = true
+  true
+
+ g) var x; x == 'undefined'
+  false
+
+ h) '9' > '10'
+  true
+
+ i) typeof x + 1 === 'number'
+  false
+
+ j) typeof x % 2 === 'number'
+  false
+
+ k) typeof (x % 2) === 'number'
+  true
+
+ l) x++ == ++x //having trouble understanding the increment operator...
+
+ m) ++x == x++ //see above
+
+ n) "1"+x == 1+x
+  false, unless x is a string
+
+ o) "0" +1 == 1
+ true
+
+ p) (typeof (x+1)) === (typeof x)
+  true, if x is a number
```

Comment:
In fact, even if x is a string, like "Bobby Brown" this will still result in true. The left side of the equation will concatenate string + 1 turning it into "Bobby Brown1" and then give the typeof, which is 'string'. 'string' will then be evaluated against (typeof 'x') which is again 'string', resulting in true.